drake/systems/plants/RigidBodySystem.cpp

```diff
@@ -367,8 +361,53 @@ RigidBodySpringDamper::RigidBodySpringDamper(RigidBodySystem* sys,
   tree->addFrame(frameB);
 }
 
+RigidBodyAccelerometer::RigidBodyAccelerometer(RigidBodySystem const& sys,
+                                               const std::string& name,
+                                               const std::shared_ptr<RigidBodyFrame> frame)
+  : RigidBodySensor(sys, name), frame(frame) {  }
+
+
+Eigen::VectorXd RigidBodyAccelerometer::output(const double &t,
+                                               const KinematicsCache<double> &rigid_body_state,
+                                               const RigidBodySystem::InputVector<double>& u) const
+{
+  auto const& q = rigid_body_state.getQ();
+  auto const& v = rigid_body_state.getV();
+  VectorXd x(q.size() + v.size());
+  x << q, v;
+  auto xdd = sys.dynamics(t, x, u);
+  auto const& tree = sys.getRigidBodyTree();
+  auto v_dot = xdd.bottomRows(v.size());
+  Vector3d sensor_origin = Vector3d::Zero(); //assumes sensor coincides with the frame's origin;
+  auto J = tree->transformPointsJacobian(rigid_body_state, sensor_origin, frame->frame_index, 0, false);
+  auto Jdot_times_v = tree->transformPointsJacobianDotTimesV(rigid_body_state, sensor_origin, frame->frame_index, 0);
+  Vector4d quat_world_to_body = tree->relativeQuaternion(rigid_body_state, 0, frame->frame_index);
+  Vector3d accel_base = Jdot_times_v + J * v_dot;
+  Vector3d accel_body = quatRotateVec(quat_world_to_body, accel_base);
+  
+  return noise_model ? noise_model->generateNoise(accel_body) : accel_body;
```

Comment:
i think we're going to want to handle this differently in the long run (so the analysis can go through without noise in a generic way for all systems, etc).  but i think it's fine for now.